apify-api/openapi/openapi.yaml

```diff
@@ -45,8 +45,8 @@ info:
 
     To use your token in a request, either:
 
-    - Add the token to your request's `Authorization` header as `Bearer
-    <token>`.
+    - Add the token to your request's `Authorization` header as <code>Bearer
+    &lt;token&gt;</code>.
```

Comment:
Is this written by hand? Are you aware of the HTML entities and the HTML tags instead of Markdown backticks?

Reply:
yes, because for some reason regular backticks are not working in new API docs check the screenshot from current version

it looks fine on Redoc version
<img width="539" alt="image" src="https://github.com/user-attachments/assets/4aff8ac5-9a20-4343-a765-a4d2e8e1093e">


but this is how it looks currently on the new API docs
<img width="548" alt="image" src="https://github.com/user-attachments/assets/75934b94-20ff-424e-a27a-ec73c0858239">


I tested this locally and it changed to backticks locally so I hope the same will happen on prod

Reply:
I'll have a look, next time ping me with such issues instead of going with some workaround, I hope we can find a way to make this work without any changes, since the workaround is ugly at best :]

Reply:
hmm but on redoc API it is rederered correctly, the issue was in new API docs?

Reply:
No, its their quirk, it makes no sense to escape anything inside code blocks. I think they just solve things on wrong level.